src/app/models/cohort-models/dimension.ts

```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright 2019  The Hyve B.V.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+export class Dimension {
+  private _name: string;
+  private _sortIndex: number;
+
+  constructor(label: string) {
+    this.name = label;
+  }
+
+  get sortIndex(): number {
```

Comment:
so the idea of the models is for glowing bear to construct VISUAL elements, except for transmart-models, if certain attributes, incl. getters and setters do not serve the purpose of visual rendering, normally we move such attributes to transmart-models, and add a mapper. (but maybe such design choice has been violated too many times...)

Reply:
it is a bit related to a visual part, because it should determine the order of dimensions in the dropdown (see one of the comments below)

Reply:
not sure how sortIndex works, how is it ordered? alphabetically? doesn't seem like so

Reply:
yes, it was not working correctly, sortIndex is removed now, sorting is fixed now and happens in the transmart-mapper